go-apps/meep-sandbox-ctrl/server/sandbox-ctrl.go

```diff
@@ -707,6 +712,54 @@ func sendEventPoasInRange(event dataModel.Event) (error, int, string) {
 	return nil, -1, description
 }
 
+// sendEventScenarioUpdate - Process a scenario update event
+func sendEventScenarioUpdate(event dataModel.Event) (error, int, string) {
+	var err error
+	var description string
+
+	if event.EventScenarioUpdate == nil {
+		err := errors.New("Malformed request: missing EventScenarioUpdate")
+		return err, http.StatusBadRequest, ""
+	}
+	if len(event.EventScenarioUpdate.Nodes) == 0 {
+		err := errors.New("Malformed request: missing Nodes")
+		return err, http.StatusBadRequest, ""
+	}
+
+	// Perform necessary action on scenario
+	switch event.EventScenarioUpdate.Action {
```

Comment:
I believe that current implementation only supports adding Physical locations of type UE
Logic to ensure that only UEs nodes are added is missing